monitoring/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '2'
+services:
+  zookeeper:
+    image: debezium/zookeeper:${DEBEZIUM_VERSION}
+    ports:
+     - 2181:2181
+     - 2888:2888
+     - 3888:3888
+  kafka:
+    image: debezium/kafka:${DEBEZIUM_VERSION}
+    ports:
+     - 9092:9092
+    links:
+     - zookeeper
+    environment:
+     - ZOOKEEPER_CONNECT=zookeeper:2181
+  sqlserver:
+    image: microsoft/mssql-server-linux:2017-CU8
+    ports:
+     - 1433:1433
+    environment:
+     - ACCEPT_EULA=Y
+     - MSSQL_PID=Standard
+     - SA_PASSWORD=Password!
+  connect:
+    build:
+      context: debezium-jmx-exporter
+    ports:
+     - 8083:8083
+     - 1976:1976
+    links:
+     - kafka
+     - sqlserver
+    environment:
+     - BOOTSTRAP_SERVERS=kafka:9092
+     - GROUP_ID=1
+     - CONFIG_STORAGE_TOPIC=my_connect_configs
+     - OFFSET_STORAGE_TOPIC=my_connect_offsets
+     - STATUS_STORAGE_TOPIC=my_connect_statuses
+     - KAFKA_OPTS=-javaagent:/kafka/jmx_prometheus_javaagent.jar=8080:/kafka/config.yml
+     - JMX_PORT=1976
```

Comment:
Who is the consumer of this env var? For Debezium's own [JMX configuration](https://github.com/debezium/docker-images/blob/master/connect-base/0.9/docker-entrypoint.sh#L78-L83), it'd have to be `JMXPORT` (no underscore), so I'm trying to figure where that is handled.

Reply:
`JMX-PORT` is used by Kafka scripts - see `kafka-run-class.sh`. I'd say that Debezium's option is superfluous.

Reply:
Interesting; wondering why it was added then. Perhaps it didn't exist in KC itself back then? Anyways, should be good.